### PR TITLE
Fixes error on transaction/to/storage

### LINKED ISSUE #82: storage returning errors for transaction -> to accounts
When storage is called on an account generated by "transaction -> to" it returns an error message "Provided address \"null\" is invalid, the capitalization checksum test failed, or its an indrect IBAN address which can't be converted." and then the data itself.

Example query: 
`{
 block(number: 5000000) {
   transactions {
     to {
       storage {
         value(at: 0)
       }
     }
   }
 }
}`

### PR DESCRIPTION
**Pull request by jonathanBuhler**
Fixes #82 and adds a test case testing null entries.

### CODE REVIEW

File: src/model/core/EthqlStorage.ts [hunk 1]
```diff
@@ -74,8 +74,11 @@ class EthqlStorage {
 
     const valueCall = { query: args.at.toString(), storageType: this.previousType };
     const rtn = this.newBase(valueCall, tempBase, web3);
-
-    return web3.eth.getStorageAt(this.address, rtn);
+    if (this.address == null) {
```
Comment (akhila-raju) [inline src/model/core/EthqlStorage.ts]: Should be "=== null"

**Comment by raulk** [comment]
@jonathanBuhler – how about simplifying this to return `null` from the `EthqlAccount#storage()` function if `this.address` is null? You could do that with a shorthand I believe:

```
return this.address && new EthqlStorage(this.address)
```

GraphQL automatically handles nulls when returned at any point in the tree.

File: src/model/core/EthqlStorage.ts [hunk 2]
```diff
@@ -105,6 +108,8 @@ class EthqlStorage {
       return web3.utils.toHex(
         web3.utils.toBN(web3.utils.sha3('0x' + this.pad(tempBase, web3))).add(web3.utils.toBN(step.query)),
       );
+    } else {
+      return step.query;
```
Comment (akhila-raju) [inline src/model/core/EthqlStorage.ts]: Why was this added?

Comment (jonathanBuhler) [reply src/model/core/EthqlStorage.ts]: Function was returning null if there was no matching storageType. Now it returns the query it was given. Changed for completeness more than anything. Doesn't change functionality.

**Pull request merged**